### ISSUE TITLE
[BUG] typescript-angular generates index.ts with wrong paths if apiPackage/modelPackage specified

### ISSUE THREAD

**Issue by kuzmeo** [OpenAPITools/openapi-generator#12005]
#### Bug Report Checklist

- [x ] Have you provided a full/minimal spec to reproduce the issue?
- [- ] Have you validated the input using an OpenAPI validator ([example](https://apidevtools.org/swagger-parser/online/))?
- [- ] Have you [tested with the latest master](https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-to-test-with-the-latest-master-of-openapi-generator) to confirm the issue still exists?
- [x ] Have you searched for related issues/PRs?
- [x ] What's the actual output vs expected output?
- [- ] [Optional] Sponsorship to speed up the bug fix or feature request ([example](https://github.com/OpenAPITools/openapi-generator/issues/6178))

##### Description

Hello!
I use gradle plugin, and when apiPackage and/or modelPackage are set, typescript-angular engine still generates index.ts with default paths instead of specified.

##### openapi-generator version

5.4.0

##### OpenAPI declaration file content or url

Can be reproduced with any OpenAPI declaration file

##### Generation Details

No specific generation details

##### Steps to reproduce

 1. Create gradle plugin configuration and specify modelPackage and apiPackage there. 
 Example:
 ```
 openApiGenerate {
    generatorName = "typescript-angular"
    inputSpec = "$projectDir/rest-api-spec.json"
    outputDir = "$projectDir/src/app/api"
    apiPackage = "services"
    modelPackage = "models"
    typeMappings = [set: "Array"]

    configOptions = [
        ngVersion        : "12.2.0",
        enumNameSuffix   : "",
        fileNaming       : "kebab-case",
        modelFileSuffix  : ".model",
        serviceFileSuffix: ".service",
        stringEnums      : "true"
    ]

}
 ```
 2. Run openApiGenerate gradle task and check that model and api paths are as specified:
 ![image](https://user-images.githubusercontent.com/87804617/160831569-230d60b6-5dbb-4e18-98bb-c4b11dd40b34.png)
 
3. Check generated index.ts.
Actual content is:
![image](https://user-images.githubusercontent.com/87804617/160831704-d6a99429-a84e-4c00-95d7-d76a15886bb1.png)
Note: api and model are default paths!

Expected content is (for provided gradle plugin example):
```
export * from './services';
export * from './models';
export * from './variables';
export * from './configuration';
export * from './api.module';
```

##### Related issues/PRs

Didn't found.

**Comment by HexagonSun** [comment]
The imports are indeed hardcoded, see e.g. [index.mustache](https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/typescript-angular/index.mustache): 
```
export * from './api/api';
export * from './model/models';
…
```

As far as I can see, this is not a bug – but the options `apiPackage` and `modelPackage` that you mention are simply not supported by the typescript-angular module, see [typescript-angular.md](https://github.com/OpenAPITools/openapi-generator/blob/master/docs/generators/typescript-angular.md) (it is currently not supported by any of the typescript generators).
I don't know how easy or hard it would be to add support for this, or if it is even desired to add it just for typescript-angular and not the others of the same family.

**Comment by raphael-jungers** [comment]
For those looking for a workaround, add an index.mustache file in your templates with this content :

```
export * from './{{apiPackage}}/api';
export * from './{{modelPackage}}/models';
export * from './variables';
export * from './configuration';
export * from './api.module';
export * from './param';
```

**Comment by bbortt** [comment]
this has been fixed for `7.0.0-beta` and should be closed imo.

https://github.com/OpenAPITools/openapi-generator/commit/dcc53c0d53b1218d32954dd3a11664093429fcce#diff-a3650fcc802279129e34afb8ca0e03210ed2c3a2f2abc27a360718346e552805.
